bears/general/QuotesBear.py

```diff
@@ -56,7 +56,8 @@ def correct_single_line_str(self, filename, file, sourcerange,
         diff.change_line(sourcerange.start.line,
                          file[sourcerange.start.line - 1],
                          replacement)
-        yield Result(self, 'You do not use the preferred quotation marks.',
+        yield Result(self, 'You do not use the preferred quotation marks.'
+                     + ' Preferred mark: ' + preferred_quotation,
```

Comment:
the strings on the two lines are not aligned together.

Python automatically concatenates strings.

And please use `%s` or `{0}` variable substitution instead of `+`